app/models/supported_file_types.rb

```diff
@@ -11,4 +11,8 @@ class SupportedFileTypes
     application/msword application/vnd.openxmlformats-officedocument.wordprocessingml.document
     application/rtf image/jpeg image/bmp image/png image/tiff application/pdf
   ].freeze
+  FURTHER_INFORMATION = %w[
```

Comment:
nit: All these constants are identical. And I think they're always going to be identical - we have one set of rules about what files we accept, and we apply it everywhere. It might be we just need one `SUPPORTED_FILE_TYPES` constant.

Reply:
yeah I kept making new ones purely if they changed in the future but I think you're right